src/com/griddynamics/cinema/CinemaCli.java

```diff
@@ -0,0 +1,134 @@
+package com.griddynamics.cinema;
+
+import java.util.Scanner;
+
+public class CinemaCli {
+    
+    private static final int ONE_HUNDRED_PERCENT = 100;
+    
+    private static final int LARGE_CINEMA_HALL_MINIMUM_SEATS_AMOUNT = 60;
+    private static final int TICKET_FULL_PRICE = 10;
+    private static final int TICKET_DISCOUNT_PRICE = 8;
+    
+    private final Scanner scanner = new Scanner(System.in);
+    
+    private final boolean isLargeCinemaHall;
+    
+    private final int amountOfRows;
```

Comment:
Redundant variable, you have `cinema`, which has that information.
Please think about others fields in this context

Reply:
Got it!